trite-osprey/src/Cart.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+
+class Cart extends React.Component {
+
+
+  render() {
+
+    return(
+      <div>
+
+        <img className='cart' src={process.env.PUBLIC_URL + `/images/cart.jpg`} style={{width: 80, height: 80}} alt="broken"/>
```

Comment:
It looks like you're following some sort of guide or maybe using a framework here that I have not seen before. So you might want to ignore this comment but… using `process.env.PUBLIC_URL` all over the place is weird when you should be able to just say `src="/images/cart.jpg"`.